{
    "version": "https://jsonfeed.org/version/1",
    "title": "Security Advisory for Python packages hosted at PyPI.org",
    "home_page_url": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip",
    "feed_url": "https://azu.github.io/github-advisory-database-rss/pip.json",
    "description": "Security Advisory for Python packages hosted at PyPI.org on GitHub",
    "items": [
        {
            "content_html": "<p>A flaw was found in Ansible 2.7.17 and prior, 2.8.9 and prior, and 2.9.6 and prior when using the Extract-Zip function from the win_unzip module as the extracted file(s) are not checked if they belong to the destination folder. An attacker could take advantage of this flaw by crafting an archive anywhere in the file system, using a path traversal. This issue is fixed in 2.10.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1737\">https://nvd.nist.gov/vuln/detail/CVE-2020-1737</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67795\">https://github.com/ansible/ansible/issues/67795</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1737\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1737</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/67799\">https://github.com/ansible/ansible/pull/67799</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/1de638b4d38d6d916588e2ad48d01f90dab8c36d\">https://github.com/samdoran/ansible/commit/1de638b4d38d6d916588e2ad48d01f90dab8c36d</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/aaf549d7870b8687209a3282841b59207735b676\">https://github.com/samdoran/ansible/commit/aaf549d7870b8687209a3282841b59207735b676</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/b60aa26e2313a8d52c0e0d3fd01696e797605b72\">https://github.com/samdoran/ansible/commit/b60aa26e2313a8d52c0e0d3fd01696e797605b72</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-893h-35v4-mxqx\">https://github.com/advisories/GHSA-893h-35v4-mxqx</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-9.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-9.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-893h-35v4-mxqx",
            "title": "[ansible] Path Traversal in Ansible",
            "date_modified": "2024-09-06T17:59:42.000Z",
            "date_published": "2021-04-20T16:43:33.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-893h-35v4-mxqx"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible 2.7.17 and prior, 2.8.9 and prior, and 2.9.6 and prior when using the Extract-Zip function from the win_unzip module as the extracted file(s) are not checked if they belong to the destination folder. An attacker could take advantage of this flaw by crafting an archive anywhere in the file system, using a path traversal. This issue is fixed in 2.10.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1737\">https://nvd.nist.gov/vuln/detail/CVE-2020-1737</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67795\">https://github.com/ansible/ansible/issues/67795</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1737\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1737</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/67799\">https://github.com/ansible/ansible/pull/67799</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/1de638b4d38d6d916588e2ad48d01f90dab8c36d\">https://github.com/samdoran/ansible/commit/1de638b4d38d6d916588e2ad48d01f90dab8c36d</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/aaf549d7870b8687209a3282841b59207735b676\">https://github.com/samdoran/ansible/commit/aaf549d7870b8687209a3282841b59207735b676</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/b60aa26e2313a8d52c0e0d3fd01696e797605b72\">https://github.com/samdoran/ansible/commit/b60aa26e2313a8d52c0e0d3fd01696e797605b72</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-893h-35v4-mxqx\">https://github.com/advisories/GHSA-893h-35v4-mxqx</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-9.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-9.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-893h-35v4-mxqx",
            "title": "[ansible] Path Traversal in Ansible",
            "date_modified": "2024-09-06T17:59:42.000Z",
            "date_published": "2021-04-20T16:43:33.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-893h-35v4-mxqx"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible 2.7.17 and prior, 2.8.9 and prior, and 2.9.6 and prior when using the Extract-Zip function from the win_unzip module as the extracted file(s) are not checked if they belong to the destination folder. An attacker could take advantage of this flaw by crafting an archive anywhere in the file system, using a path traversal. This issue is fixed in 2.10.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1737\">https://nvd.nist.gov/vuln/detail/CVE-2020-1737</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67795\">https://github.com/ansible/ansible/issues/67795</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1737\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1737</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/67799\">https://github.com/ansible/ansible/pull/67799</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/1de638b4d38d6d916588e2ad48d01f90dab8c36d\">https://github.com/samdoran/ansible/commit/1de638b4d38d6d916588e2ad48d01f90dab8c36d</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/aaf549d7870b8687209a3282841b59207735b676\">https://github.com/samdoran/ansible/commit/aaf549d7870b8687209a3282841b59207735b676</a></li>\n<li><a href=\"https://github.com/samdoran/ansible/commit/b60aa26e2313a8d52c0e0d3fd01696e797605b72\">https://github.com/samdoran/ansible/commit/b60aa26e2313a8d52c0e0d3fd01696e797605b72</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-893h-35v4-mxqx\">https://github.com/advisories/GHSA-893h-35v4-mxqx</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-9.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-9.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-893h-35v4-mxqx",
            "title": "[ansible] Path Traversal in Ansible",
            "date_modified": "2024-09-06T17:59:42.000Z",
            "date_published": "2021-04-20T16:43:33.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-893h-35v4-mxqx"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible 2.7.16 and prior, 2.8.8 and prior, and 2.9.5 and prior when a password is set with the argument \"password\" of svn module, it is used on svn command line, disclosing to other users within the same node. An attacker could take advantage by reading the cmdline file from that particular PID on the procfs.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1739\">https://nvd.nist.gov/vuln/detail/CVE-2020-1739</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67797\">https://github.com/ansible/ansible/issues/67797</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1739\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1739</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html\">https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68911\">https://github.com/ansible/ansible/pull/68911</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68912\">https://github.com/ansible/ansible/pull/68912</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68913\">https://github.com/ansible/ansible/pull/68913</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/1a89d4f059c21a818306a39ada7f5284ae125237\">https://github.com/ansible/ansible/commit/1a89d4f059c21a818306a39ada7f5284ae125237</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/6c74a298702c8bb5532b9600073312e08f39680f\">https://github.com/ansible/ansible/commit/6c74a298702c8bb5532b9600073312e08f39680f</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/c6c4fbf4a1fdea1e10ba94462a60c413990a16a4\">https://github.com/ansible/ansible/commit/c6c4fbf4a1fdea1e10ba94462a60c413990a16a4</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-923p-fr2c-g5m2\">https://github.com/advisories/GHSA-923p-fr2c-g5m2</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-11.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-11.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-923p-fr2c-g5m2",
            "title": "[ansible] Exposure of Sensitive Information to an Unauthorized Actor in Ansible",
            "date_modified": "2024-09-06T17:54:00.000Z",
            "date_published": "2021-04-07T20:30:44.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-923p-fr2c-g5m2"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible 2.7.16 and prior, 2.8.8 and prior, and 2.9.5 and prior when a password is set with the argument \"password\" of svn module, it is used on svn command line, disclosing to other users within the same node. An attacker could take advantage by reading the cmdline file from that particular PID on the procfs.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1739\">https://nvd.nist.gov/vuln/detail/CVE-2020-1739</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67797\">https://github.com/ansible/ansible/issues/67797</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1739\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1739</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html\">https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68911\">https://github.com/ansible/ansible/pull/68911</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68912\">https://github.com/ansible/ansible/pull/68912</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68913\">https://github.com/ansible/ansible/pull/68913</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/1a89d4f059c21a818306a39ada7f5284ae125237\">https://github.com/ansible/ansible/commit/1a89d4f059c21a818306a39ada7f5284ae125237</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/6c74a298702c8bb5532b9600073312e08f39680f\">https://github.com/ansible/ansible/commit/6c74a298702c8bb5532b9600073312e08f39680f</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/c6c4fbf4a1fdea1e10ba94462a60c413990a16a4\">https://github.com/ansible/ansible/commit/c6c4fbf4a1fdea1e10ba94462a60c413990a16a4</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-923p-fr2c-g5m2\">https://github.com/advisories/GHSA-923p-fr2c-g5m2</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-11.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-11.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-923p-fr2c-g5m2",
            "title": "[ansible] Exposure of Sensitive Information to an Unauthorized Actor in Ansible",
            "date_modified": "2024-09-06T17:54:00.000Z",
            "date_published": "2021-04-07T20:30:44.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-923p-fr2c-g5m2"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible 2.7.16 and prior, 2.8.8 and prior, and 2.9.5 and prior when a password is set with the argument \"password\" of svn module, it is used on svn command line, disclosing to other users within the same node. An attacker could take advantage by reading the cmdline file from that particular PID on the procfs.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1739\">https://nvd.nist.gov/vuln/detail/CVE-2020-1739</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67797\">https://github.com/ansible/ansible/issues/67797</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1739\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1739</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html\">https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68911\">https://github.com/ansible/ansible/pull/68911</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68912\">https://github.com/ansible/ansible/pull/68912</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/68913\">https://github.com/ansible/ansible/pull/68913</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/1a89d4f059c21a818306a39ada7f5284ae125237\">https://github.com/ansible/ansible/commit/1a89d4f059c21a818306a39ada7f5284ae125237</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/6c74a298702c8bb5532b9600073312e08f39680f\">https://github.com/ansible/ansible/commit/6c74a298702c8bb5532b9600073312e08f39680f</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/c6c4fbf4a1fdea1e10ba94462a60c413990a16a4\">https://github.com/ansible/ansible/commit/c6c4fbf4a1fdea1e10ba94462a60c413990a16a4</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-923p-fr2c-g5m2\">https://github.com/advisories/GHSA-923p-fr2c-g5m2</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-11.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-11.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/FWDK3QUVBULS3Q3PQTGEKUQYPSNOU5M3</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/QT27K5ZRGDPCH7GT3DRI3LO4IVDVQUB7</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/U3IMV3XEIUXL6S4KPLYYM4TVJQ2VNEP2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-923p-fr2c-g5m2",
            "title": "[ansible] Exposure of Sensitive Information to an Unauthorized Actor in Ansible",
            "date_modified": "2024-09-06T17:54:00.000Z",
            "date_published": "2021-04-07T20:30:44.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-923p-fr2c-g5m2"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible Engine, all versions 2.7.x, 2.8.x and 2.9.x prior to 2.7.17, 2.8.11, and 2.9.7 respectively, when using ansible_facts as a subkey of itself and promoting it to a variable when inject is enabled, overwriting the ansible_facts after the clean. An attacker could take advantage of this by altering the ansible_facts, such as ansible_hosts, users and any other key data which would lead into privilege escalation or code injection.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-10684\">https://nvd.nist.gov/vuln/detail/CVE-2020-10684</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-10684\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-10684</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/a9d2ceafe429171c0e2ad007058b88bae57c74ce\">https://github.com/ansible/ansible/commit/a9d2ceafe429171c0e2ad007058b88bae57c74ce</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/0b4788a71fc7d24ffa957a94ee5e23d6a9733ab0\">https://github.com/ansible/ansible/commit/0b4788a71fc7d24ffa957a94ee5e23d6a9733ab0</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/1d0d2645eed36ac4e17052ab4eacf240132d96fb\">https://github.com/ansible/ansible/commit/1d0d2645eed36ac4e17052ab4eacf240132d96fb</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/5eabf7bb93c9bfc375b806a2b1f623d650cddc2b\">https://github.com/ansible/ansible/commit/5eabf7bb93c9bfc375b806a2b1f623d650cddc2b</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-p62g-jhg6-v3rq\">https://github.com/advisories/GHSA-p62g-jhg6-v3rq</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-207.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-207.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-p62g-jhg6-v3rq",
            "title": "[ansible] Code Injection, Race Condition, and Execution with Unnecessary Privileges in Ansible",
            "date_modified": "2024-09-06T17:49:53.000Z",
            "date_published": "2021-04-07T20:37:06.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-p62g-jhg6-v3rq"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible Engine, all versions 2.7.x, 2.8.x and 2.9.x prior to 2.7.17, 2.8.11, and 2.9.7 respectively, when using ansible_facts as a subkey of itself and promoting it to a variable when inject is enabled, overwriting the ansible_facts after the clean. An attacker could take advantage of this by altering the ansible_facts, such as ansible_hosts, users and any other key data which would lead into privilege escalation or code injection.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-10684\">https://nvd.nist.gov/vuln/detail/CVE-2020-10684</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-10684\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-10684</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/a9d2ceafe429171c0e2ad007058b88bae57c74ce\">https://github.com/ansible/ansible/commit/a9d2ceafe429171c0e2ad007058b88bae57c74ce</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/0b4788a71fc7d24ffa957a94ee5e23d6a9733ab0\">https://github.com/ansible/ansible/commit/0b4788a71fc7d24ffa957a94ee5e23d6a9733ab0</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/1d0d2645eed36ac4e17052ab4eacf240132d96fb\">https://github.com/ansible/ansible/commit/1d0d2645eed36ac4e17052ab4eacf240132d96fb</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/5eabf7bb93c9bfc375b806a2b1f623d650cddc2b\">https://github.com/ansible/ansible/commit/5eabf7bb93c9bfc375b806a2b1f623d650cddc2b</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-p62g-jhg6-v3rq\">https://github.com/advisories/GHSA-p62g-jhg6-v3rq</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-207.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-207.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-p62g-jhg6-v3rq",
            "title": "[ansible] Code Injection, Race Condition, and Execution with Unnecessary Privileges in Ansible",
            "date_modified": "2024-09-06T17:49:53.000Z",
            "date_published": "2021-04-07T20:37:06.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-p62g-jhg6-v3rq"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible Engine, all versions 2.7.x, 2.8.x and 2.9.x prior to 2.7.17, 2.8.11, and 2.9.7 respectively, when using ansible_facts as a subkey of itself and promoting it to a variable when inject is enabled, overwriting the ansible_facts after the clean. An attacker could take advantage of this by altering the ansible_facts, such as ansible_hosts, users and any other key data which would lead into privilege escalation or code injection.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-10684\">https://nvd.nist.gov/vuln/detail/CVE-2020-10684</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-10684\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-10684</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/a9d2ceafe429171c0e2ad007058b88bae57c74ce\">https://github.com/ansible/ansible/commit/a9d2ceafe429171c0e2ad007058b88bae57c74ce</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/0b4788a71fc7d24ffa957a94ee5e23d6a9733ab0\">https://github.com/ansible/ansible/commit/0b4788a71fc7d24ffa957a94ee5e23d6a9733ab0</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/1d0d2645eed36ac4e17052ab4eacf240132d96fb\">https://github.com/ansible/ansible/commit/1d0d2645eed36ac4e17052ab4eacf240132d96fb</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/5eabf7bb93c9bfc375b806a2b1f623d650cddc2b\">https://github.com/ansible/ansible/commit/5eabf7bb93c9bfc375b806a2b1f623d650cddc2b</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-p62g-jhg6-v3rq\">https://github.com/advisories/GHSA-p62g-jhg6-v3rq</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-207.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-207.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-p62g-jhg6-v3rq",
            "title": "[ansible] Code Injection, Race Condition, and Execution with Unnecessary Privileges in Ansible",
            "date_modified": "2024-09-06T17:49:53.000Z",
            "date_published": "2021-04-07T20:37:06.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-p62g-jhg6-v3rq"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible, where a user's controller is vulnerable to template injection. This issue can occur through facts used in the template if the user is trying to put templates in multi-line YAML strings and the facts being handled do not routinely include special template characters. This flaw allows attackers to perform command injection, which discloses sensitive information. The highest threat from this vulnerability is to confidentiality and integrity.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3583\">https://nvd.nist.gov/vuln/detail/CVE-2021-3583</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/74960\">https://github.com/ansible/ansible/pull/74960</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1968412\">https://bugzilla.redhat.com/show_bug.cgi?id=1968412</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2023/12/msg00018.html\">https://lists.debian.org/debian-lts-announce/2023/12/msg00018.html</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/03aff644cc1c00e1f7551195c68fbd0d13a39e6e\">https://github.com/ansible/ansible/commit/03aff644cc1c00e1f7551195c68fbd0d13a39e6e</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8aa850e3573e48c9a2f12aef84e8a3a6f5ba4847\">https://github.com/ansible/ansible/commit/8aa850e3573e48c9a2f12aef84e8a3a6f5ba4847</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8b17e5b9229ffaecfe10a4881bc3f87dd2c184e1\">https://github.com/ansible/ansible/commit/8b17e5b9229ffaecfe10a4881bc3f87dd2c184e1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-2pfh-q76x-gwvm\">https://github.com/advisories/GHSA-2pfh-q76x-gwvm</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-358.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-358.yaml</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-2pfh-q76x-gwvm",
            "title": "[ansible] Improper Input Validation and Command Injection in Ansible",
            "date_modified": "2024-09-06T17:47:40.000Z",
            "date_published": "2021-09-23T23:16:38.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-2pfh-q76x-gwvm"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible, where a user's controller is vulnerable to template injection. This issue can occur through facts used in the template if the user is trying to put templates in multi-line YAML strings and the facts being handled do not routinely include special template characters. This flaw allows attackers to perform command injection, which discloses sensitive information. The highest threat from this vulnerability is to confidentiality and integrity.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3583\">https://nvd.nist.gov/vuln/detail/CVE-2021-3583</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/74960\">https://github.com/ansible/ansible/pull/74960</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1968412\">https://bugzilla.redhat.com/show_bug.cgi?id=1968412</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2023/12/msg00018.html\">https://lists.debian.org/debian-lts-announce/2023/12/msg00018.html</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/03aff644cc1c00e1f7551195c68fbd0d13a39e6e\">https://github.com/ansible/ansible/commit/03aff644cc1c00e1f7551195c68fbd0d13a39e6e</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8aa850e3573e48c9a2f12aef84e8a3a6f5ba4847\">https://github.com/ansible/ansible/commit/8aa850e3573e48c9a2f12aef84e8a3a6f5ba4847</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8b17e5b9229ffaecfe10a4881bc3f87dd2c184e1\">https://github.com/ansible/ansible/commit/8b17e5b9229ffaecfe10a4881bc3f87dd2c184e1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-2pfh-q76x-gwvm\">https://github.com/advisories/GHSA-2pfh-q76x-gwvm</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-358.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-358.yaml</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-2pfh-q76x-gwvm",
            "title": "[ansible] Improper Input Validation and Command Injection in Ansible",
            "date_modified": "2024-09-06T17:47:40.000Z",
            "date_published": "2021-09-23T23:16:38.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-2pfh-q76x-gwvm"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in Ansible, where a user's controller is vulnerable to template injection. This issue can occur through facts used in the template if the user is trying to put templates in multi-line YAML strings and the facts being handled do not routinely include special template characters. This flaw allows attackers to perform command injection, which discloses sensitive information. The highest threat from this vulnerability is to confidentiality and integrity.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-3583\">https://nvd.nist.gov/vuln/detail/CVE-2021-3583</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/74960\">https://github.com/ansible/ansible/pull/74960</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1968412\">https://bugzilla.redhat.com/show_bug.cgi?id=1968412</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2023/12/msg00018.html\">https://lists.debian.org/debian-lts-announce/2023/12/msg00018.html</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/03aff644cc1c00e1f7551195c68fbd0d13a39e6e\">https://github.com/ansible/ansible/commit/03aff644cc1c00e1f7551195c68fbd0d13a39e6e</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8aa850e3573e48c9a2f12aef84e8a3a6f5ba4847\">https://github.com/ansible/ansible/commit/8aa850e3573e48c9a2f12aef84e8a3a6f5ba4847</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8b17e5b9229ffaecfe10a4881bc3f87dd2c184e1\">https://github.com/ansible/ansible/commit/8b17e5b9229ffaecfe10a4881bc3f87dd2c184e1</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-2pfh-q76x-gwvm\">https://github.com/advisories/GHSA-2pfh-q76x-gwvm</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-358.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-358.yaml</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-2pfh-q76x-gwvm",
            "title": "[ansible] Improper Input Validation and Command Injection in Ansible",
            "date_modified": "2024-09-06T17:47:40.000Z",
            "date_published": "2021-09-23T23:16:38.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-2pfh-q76x-gwvm"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A race condition flaw was found in Ansible Engine 2.7.17 and prior, 2.8.9 and prior, 2.9.6 and prior when running a playbook with an unprivileged become user. When Ansible needs to run a module with become user, the temporary directory is created in /var/tmp. This directory is created with \"umask 77 &amp;&amp; mkdir -p </p><dir>\"; this operation does not fail if the directory already exists and is owned by another user. An attacker could take advantage to gain control of the become user as the target directory can be retrieved by iterating '/proc//cmdline'.<p></p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1733\">https://nvd.nist.gov/vuln/detail/CVE-2020-1733</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67791\">https://github.com/ansible/ansible/issues/67791</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1733\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1733</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html\">https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/80b9a0a25c5f75e84aefc8f2b293fb1933b154f2\">https://github.com/ansible/ansible/commit/80b9a0a25c5f75e84aefc8f2b293fb1933b154f2</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8251d9f4c2bc82632ab992277fcd30ccbf87aa47\">https://github.com/ansible/ansible/commit/8251d9f4c2bc82632ab992277fcd30ccbf87aa47</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/ecf99d5e1ff732a7777010facd6c98bb0994605e\">https://github.com/ansible/ansible/commit/ecf99d5e1ff732a7777010facd6c98bb0994605e</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-g4mq-6fp5-qwcf\">https://github.com/advisories/GHSA-g4mq-6fp5-qwcf</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-5.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB</a></li>\n</ul>\n</dir>",
            "url": "https://github.com/advisories/GHSA-g4mq-6fp5-qwcf",
            "title": "[ansible] Ansible vulnerable to Exposure of Resource to Wrong Sphere and Insecure Temporary File",
            "date_modified": "2024-09-06T17:42:35.000Z",
            "date_published": "2021-04-20T16:46:12.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-g4mq-6fp5-qwcf"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A race condition flaw was found in Ansible Engine 2.7.17 and prior, 2.8.9 and prior, 2.9.6 and prior when running a playbook with an unprivileged become user. When Ansible needs to run a module with become user, the temporary directory is created in /var/tmp. This directory is created with \"umask 77 &amp;&amp; mkdir -p </p><dir>\"; this operation does not fail if the directory already exists and is owned by another user. An attacker could take advantage to gain control of the become user as the target directory can be retrieved by iterating '/proc//cmdline'.<p></p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1733\">https://nvd.nist.gov/vuln/detail/CVE-2020-1733</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67791\">https://github.com/ansible/ansible/issues/67791</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1733\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1733</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html\">https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/80b9a0a25c5f75e84aefc8f2b293fb1933b154f2\">https://github.com/ansible/ansible/commit/80b9a0a25c5f75e84aefc8f2b293fb1933b154f2</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8251d9f4c2bc82632ab992277fcd30ccbf87aa47\">https://github.com/ansible/ansible/commit/8251d9f4c2bc82632ab992277fcd30ccbf87aa47</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/ecf99d5e1ff732a7777010facd6c98bb0994605e\">https://github.com/ansible/ansible/commit/ecf99d5e1ff732a7777010facd6c98bb0994605e</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-g4mq-6fp5-qwcf\">https://github.com/advisories/GHSA-g4mq-6fp5-qwcf</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-5.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB</a></li>\n</ul>\n</dir>",
            "url": "https://github.com/advisories/GHSA-g4mq-6fp5-qwcf",
            "title": "[ansible] Ansible vulnerable to Exposure of Resource to Wrong Sphere and Insecure Temporary File",
            "date_modified": "2024-09-06T17:42:35.000Z",
            "date_published": "2021-04-20T16:46:12.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-g4mq-6fp5-qwcf"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A race condition flaw was found in Ansible Engine 2.7.17 and prior, 2.8.9 and prior, 2.9.6 and prior when running a playbook with an unprivileged become user. When Ansible needs to run a module with become user, the temporary directory is created in /var/tmp. This directory is created with \"umask 77 &amp;&amp; mkdir -p </p><dir>\"; this operation does not fail if the directory already exists and is owned by another user. An attacker could take advantage to gain control of the become user as the target directory can be retrieved by iterating '/proc//cmdline'.<p></p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-1733\">https://nvd.nist.gov/vuln/detail/CVE-2020-1733</a></li>\n<li><a href=\"https://github.com/ansible/ansible/issues/67791\">https://github.com/ansible/ansible/issues/67791</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1733\">https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-1733</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html\">https://lists.debian.org/debian-lts-announce/2020/05/msg00005.html</a></li>\n<li><a href=\"https://security.gentoo.org/glsa/202006-11\">https://security.gentoo.org/glsa/202006-11</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/80b9a0a25c5f75e84aefc8f2b293fb1933b154f2\">https://github.com/ansible/ansible/commit/80b9a0a25c5f75e84aefc8f2b293fb1933b154f2</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/8251d9f4c2bc82632ab992277fcd30ccbf87aa47\">https://github.com/ansible/ansible/commit/8251d9f4c2bc82632ab992277fcd30ccbf87aa47</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/ecf99d5e1ff732a7777010facd6c98bb0994605e\">https://github.com/ansible/ansible/commit/ecf99d5e1ff732a7777010facd6c98bb0994605e</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-g4mq-6fp5-qwcf\">https://github.com/advisories/GHSA-g4mq-6fp5-qwcf</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-5.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2020-5.yaml</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/DKPA4KC3OJSUFASUYMG66HKJE7ADNGFW</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/MRRYUU5ZBLPBXCYG6CFP35D64NP2UB2S</a></li>\n<li><a href=\"https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB\">https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/WQVOQD4VAIXXTVQAJKTN7NUGTJFE2PCB</a></li>\n</ul>\n</dir>",
            "url": "https://github.com/advisories/GHSA-g4mq-6fp5-qwcf",
            "title": "[ansible] Ansible vulnerable to Exposure of Resource to Wrong Sphere and Insecure Temporary File",
            "date_modified": "2024-09-06T17:42:35.000Z",
            "date_published": "2021-04-20T16:46:12.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-g4mq-6fp5-qwcf"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>CKAN through 2.9.6 account takeovers by unauthenticated users when an existing user id is sent via an HTTP POST request. This allows a user to take over an existing account including superuser accounts.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2022-43685\">https://nvd.nist.gov/vuln/detail/CVE-2022-43685</a></li>\n<li><a href=\"https://ckan.org/blog/get-latest-patch-releases-your-ckan-site-october-2022\">https://ckan.org/blog/get-latest-patch-releases-your-ckan-site-october-2022</a></li>\n<li><a href=\"https://ckan.org\">https://ckan.org</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ckan/PYSEC-2022-42987.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ckan/PYSEC-2022-42987.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-m2xp-jxfg-qq6g\">https://github.com/advisories/GHSA-m2xp-jxfg-qq6g</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-m2xp-jxfg-qq6g",
            "title": "[ckan] CKAN contains Improper Authentication leading to account takeover",
            "date_modified": "2024-09-06T16:31:22.000Z",
            "date_published": "2022-11-22T03:30:56.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-m2xp-jxfg-qq6g"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Cross Site Scripting (XSS) vulnerability in Arachnys Cabot up to and including 0.11.12 can be exploited via the Address column.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-25449\">https://nvd.nist.gov/vuln/detail/CVE-2020-25449</a></li>\n<li><a href=\"https://itsmeanonartist.tech/blogs/blog2.html\">https://itsmeanonartist.tech/blogs/blog2.html</a></li>\n<li><a href=\"https://packetstormsecurity.com/files/159070/Cabot-0.11.12-Cross-Site-Scripting.html\">https://packetstormsecurity.com/files/159070/Cabot-0.11.12-Cross-Site-Scripting.html</a></li>\n<li><a href=\"https://www.exploit-db.com/exploits/48791\">https://www.exploit-db.com/exploits/48791</a></li>\n<li><a href=\"https://www.exploitalert.com/view-details.html?id=36106\">https://www.exploitalert.com/view-details.html?id=36106</a></li>\n<li><a href=\"https://github.com/arachnys/cabot/commit/18708572cb0ed143842409419eada91160413973\">https://github.com/arachnys/cabot/commit/18708572cb0ed143842409419eada91160413973</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/cabot/PYSEC-2020-226.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/cabot/PYSEC-2020-226.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-8q2h-4mq6-396j\">https://github.com/advisories/GHSA-8q2h-4mq6-396j</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-8q2h-4mq6-396j",
            "title": "[cabot] Cabot Cross Site Scripting (XSS) vulnerability via Address column",
            "date_modified": "2024-09-06T16:30:06.000Z",
            "date_published": "2022-05-24T17:35:22.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-8q2h-4mq6-396j"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was discovered in OpenStack blazar-dashboard before 1.3.1, 2.0.0, and 3.0.0. A user allowed to access the Blazar dashboard in Horizon may trigger code execution on the Horizon host as the user the Horizon service runs under (because the Python eval function is used). This may result in Horizon host unauthorized access and further compromise of the Horizon service. All setups using the Horizon dashboard with the blazar-dashboard plugin are affected.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-26943\">https://nvd.nist.gov/vuln/detail/CVE-2020-26943</a></li>\n<li><a href=\"https://launchpad.net/bugs/1895688\">https://launchpad.net/bugs/1895688</a></li>\n<li><a href=\"https://review.opendev.org/755810\">https://review.opendev.org/755810</a></li>\n<li><a href=\"https://review.opendev.org/755812\">https://review.opendev.org/755812</a></li>\n<li><a href=\"https://review.opendev.org/755813\">https://review.opendev.org/755813</a></li>\n<li><a href=\"https://review.opendev.org/755814\">https://review.opendev.org/755814</a></li>\n<li><a href=\"https://review.opendev.org/756064\">https://review.opendev.org/756064</a></li>\n<li><a href=\"https://security.openstack.org/ossa/OSSA-2020-007.html\">https://security.openstack.org/ossa/OSSA-2020-007.html</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2020/10/16/5\">http://www.openwall.com/lists/oss-security/2020/10/16/5</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/blazar-dashboard/PYSEC-2020-225.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/blazar-dashboard/PYSEC-2020-225.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-939m-4xpw-v34v\">https://github.com/advisories/GHSA-939m-4xpw-v34v</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-939m-4xpw-v34v",
            "title": "[blazar-dashboard] Arbitrary Code Execution in blazar-dashboard",
            "date_modified": "2024-09-06T16:25:49.000Z",
            "date_published": "2020-10-27T17:55:04.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-939m-4xpw-v34v"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was discovered in OpenStack blazar-dashboard before 1.3.1, 2.0.0, and 3.0.0. A user allowed to access the Blazar dashboard in Horizon may trigger code execution on the Horizon host as the user the Horizon service runs under (because the Python eval function is used). This may result in Horizon host unauthorized access and further compromise of the Horizon service. All setups using the Horizon dashboard with the blazar-dashboard plugin are affected.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-26943\">https://nvd.nist.gov/vuln/detail/CVE-2020-26943</a></li>\n<li><a href=\"https://launchpad.net/bugs/1895688\">https://launchpad.net/bugs/1895688</a></li>\n<li><a href=\"https://review.opendev.org/755810\">https://review.opendev.org/755810</a></li>\n<li><a href=\"https://review.opendev.org/755812\">https://review.opendev.org/755812</a></li>\n<li><a href=\"https://review.opendev.org/755813\">https://review.opendev.org/755813</a></li>\n<li><a href=\"https://review.opendev.org/755814\">https://review.opendev.org/755814</a></li>\n<li><a href=\"https://review.opendev.org/756064\">https://review.opendev.org/756064</a></li>\n<li><a href=\"https://security.openstack.org/ossa/OSSA-2020-007.html\">https://security.openstack.org/ossa/OSSA-2020-007.html</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2020/10/16/5\">http://www.openwall.com/lists/oss-security/2020/10/16/5</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/blazar-dashboard/PYSEC-2020-225.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/blazar-dashboard/PYSEC-2020-225.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-939m-4xpw-v34v\">https://github.com/advisories/GHSA-939m-4xpw-v34v</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-939m-4xpw-v34v",
            "title": "[blazar-dashboard] Arbitrary Code Execution in blazar-dashboard",
            "date_modified": "2024-09-06T16:25:49.000Z",
            "date_published": "2020-10-27T17:55:04.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-939m-4xpw-v34v"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An issue was discovered in OpenStack blazar-dashboard before 1.3.1, 2.0.0, and 3.0.0. A user allowed to access the Blazar dashboard in Horizon may trigger code execution on the Horizon host as the user the Horizon service runs under (because the Python eval function is used). This may result in Horizon host unauthorized access and further compromise of the Horizon service. All setups using the Horizon dashboard with the blazar-dashboard plugin are affected.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-26943\">https://nvd.nist.gov/vuln/detail/CVE-2020-26943</a></li>\n<li><a href=\"https://launchpad.net/bugs/1895688\">https://launchpad.net/bugs/1895688</a></li>\n<li><a href=\"https://review.opendev.org/755810\">https://review.opendev.org/755810</a></li>\n<li><a href=\"https://review.opendev.org/755812\">https://review.opendev.org/755812</a></li>\n<li><a href=\"https://review.opendev.org/755813\">https://review.opendev.org/755813</a></li>\n<li><a href=\"https://review.opendev.org/755814\">https://review.opendev.org/755814</a></li>\n<li><a href=\"https://review.opendev.org/756064\">https://review.opendev.org/756064</a></li>\n<li><a href=\"https://security.openstack.org/ossa/OSSA-2020-007.html\">https://security.openstack.org/ossa/OSSA-2020-007.html</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2020/10/16/5\">http://www.openwall.com/lists/oss-security/2020/10/16/5</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/blazar-dashboard/PYSEC-2020-225.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/blazar-dashboard/PYSEC-2020-225.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-939m-4xpw-v34v\">https://github.com/advisories/GHSA-939m-4xpw-v34v</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-939m-4xpw-v34v",
            "title": "[blazar-dashboard] Arbitrary Code Execution in blazar-dashboard",
            "date_modified": "2024-09-06T16:25:49.000Z",
            "date_published": "2020-10-27T17:55:04.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-939m-4xpw-v34v"
            },
            "tags": [
                "severity"
            ]
        }
    ]
}